lms/templates/student_account/institution_login.underscore

```diff
@@ -0,0 +1,32 @@
+<div class="other-login-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text">
+                <span class="icon fa fa-university" aria-hidden="true"></span>
+                <%- gettext("Sign in with Institution/Campus Credentials") %>
+            </span>
+        </h2>
+    </div>
+
+    <p><%- gettext("Choose your institution from the list below:") %></p>
```

Comment:
Visual Feedback: This text is rendered currently in a serif typeface. Please make sure it renders in our Open Sans typeface. Our [typography Sass partial](https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss) could be your bestie for things like this - please extend a placeholder from it.